backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc

```diff
@@ -1,32 +1,45 @@
 [id="installing-oadp-ocs"]
-= Installing the Openshift API for Data Protection with OpenShift Container Storage
+= Installing and configuring the Openshift API for Data Protection with OpenShift Container Storage
 include::modules/common-attributes.adoc[]
 :context: installing-oadp-ocs
 :installing-oadp-ocs:
-:credentials: cloud-credentials-gcp
+:credentials: cloud-credentials
 :provider: gcp
 
 toc::[]
 
-You install the Openshift API for Data Protection (OADP) with OpenShift Container Storage (OCS) by installing the OADP Operator, installing the Data Protection Application, and configuring the snapshot location.
-
-To install the OADP Operator in a restricted network environment, you must first disable the default OperatorHub sources and mirror the Operator catalog. See xref:../../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for details.
+You install the Openshift API for Data Protection (OADP) with OpenShift Container Storage (OCS) by installing the OADP Operator and configuring a backup location and a snapshot location. Then, you install the Data Protection Application.
 
 You can configure xref:../../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc#installing-oadp-mcg[Multicloud Object Gateway] or any S3-compatible object storage as a backup location.
 
-You can configure xref:../../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc#oadp-about-vsl_installing-oadp-ocs[Ceph RBD or Ceph FS storage] as a snapshot location.
+:FeatureName: The `CloudStorage` API for S3 storage
+include::modules/technology-preview.adoc[]
 
-If you are not using snapshots or if your storage is NFS, you can create backups with link:https://restic.net/[Restic].
+You can configure Container Storage Interface (CSI) snapshot-enabled storage, such as Ceph RBD or Ceph FS, as a snapshot location.
```

Comment:
For CSI snapshot, it doesn't need a snapshot location DataProtectionApplication in DAP